pages/docs/tutorials/kafka/managing-schemas-using-schema-registry.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Managing schemas using Schema Registry
+description: The guide explains how you can manage schemas using Schema Registry.
```

Comment:
This is not a guide, this is **tutorial**. Please fix the description meta data accordingly.

Reply:
fixed